src/paths/ActiveCalls.ts

```diff
@@ -12,12 +12,7 @@ export default class ActiveCalls extends PathSegment {
      *  Get Account Active (Recent) Calls
      */
     list(query?: ListQuery): Promise<PagingResult<CallLogRecord>> {
-        return this.getService().send({
-          body: undefined,
-          method: "get",
-          query: query,
-          url: this.getEndpoint(false),
-        }).then((res) => {
+        return this._send("get", false, undefined, query).then((res) => {
```

Comment:
It appeared first in the source, that's why I commented here. This applies to all other methods, bc the concept should be the same, so list should also have listRaw, despite the fact that it's always json, it makes API predictable.